velox/type/Filter.cpp

```diff
@@ -143,6 +146,18 @@ std::vector<int64_t> deserializeValues(const folly::dynamic& obj) {
   }
   return values;
 }
+
+std::vector<int128_t> deserializeHugeintValues(const folly::dynamic& obj) {
+  std::vector<int128_t> values;
+  auto lowerValuesArray = obj["lower_values"];
+  auto upperValuesArray = obj["upper_values"];
+  auto len = lowerValuesArray.size();
+  values.reserve(len);
+  for (auto i = 0; i < len; i++) {
+    HugeInt::build(upperValuesArray[i].asInt(), lowerValuesArray[i].asInt());
```

Comment:
Shouldn't this be `values[i] = HugeInt::build(upperValuesArray[i].asInt(), lowerValuesArray[i].asInt());`

Can we add a test?